tests/RobotFramework/tasks/E2E_Tutorials/Connect_to_c8y.robot

```diff
@@ -26,19 +26,8 @@ Set the URL of your Cumulocity IoT tenant
     ${HOSTNAME}=      Strip String    ${HOSTNAME}    characters=/
     Execute Command    sudo tedge config set c8y.url ${HOSTNAME}    # Set the URL of your Cumulocity IoT tenant
 
-Create the certificate
-    Execute Command    sudo tedge cert create --device-id ${DEVICE_SN}
-
-    #You can then check the content of that certificate.
-    ${output}=    Execute Command    sudo tedge cert show    #You can then check the content of that certificate.
-    Should Contain    ${output}    Device certificate: /etc/tedge/device-certs/tedge-certificate.pem
-    Should Contain    ${output}    Subject: CN=${DEVICE_SN}, O=Thin Edge, OU=Test Device
-    Should Contain    ${output}    Issuer: CN=${DEVICE_SN}, O=Thin Edge, OU=Test Device
-    Should Contain    ${output}    Valid from:
-    Should Contain    ${output}    Valid up to:
-    Should Contain    ${output}    Thumbprint:
-
 tedge cert upload c8y command
```

Comment:
This test has been added here to automate all the steps of the tutorial.

Even, if duplicated elsewhere, it's better to keep it here.